<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Image Processor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="left">
        <item>
         <widget class="QLabel" name="labelSelectFiles">
          <property name="text">
           <string>Select File / Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="dirSelector"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="middle">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Results</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QProgressBar" name="progressBarLoadingFiles">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="imagesList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right">
        <item>
         <widget class="QLabel" name="labelPreview">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="imagePreview"/>
        </item>
        <item>
         <widget class="QLabel" name="labelSelectOutputDir">
          <property name="text">
           <string>Output Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelOutputDir">
            <property name="text">
             <string>/my/dir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonOutput">
            <property name="text">
             <string>Select Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelAlgo">
          <property name="text">
           <string>Algorithm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="algoSelector"/>
        </item>
        <item>
         <widget class="QLabel" name="labelDimension">
          <property name="text">
           <string>Dimension</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBoxW">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWidth">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxH">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHeight">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonConvert">
          <property name="text">
           <string>Convert Images</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="convertStatusLayout">
          <item>
           <widget class="QProgressBar" name="progressBarConvert">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonCancelConvert">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
